### PR TITLE
N8N-3666 Public API FE Telemetry

### PR DESCRIPTION
**Pull request by alexgrozav**
- Added telemetry events for create key, delete key, copy key and activate workflow.

### CODE REVIEW

File: packages/editor-ui/src/components/WorkflowActivator.vue [hunk 1]
```diff
@@ -120,6 +120,13 @@ export default mixins(
 							return;
 						}
 
+						if (newActiveState) {
+							this.$telemetry.track('User set workflow active status', {
+								user_id: this.$store.getters['users/currentUserId'],
+								instance_id: this.$store.getters.instanceId,
```
Comment (mutdmour) [inline packages/editor-ui/src/components/WorkflowActivator.vue:126]: you don't need to add these. these are included by default

File: packages/editor-ui/src/views/SettingsApiView.vue [hunk 2]
```diff
@@ -115,6 +120,11 @@ export default mixins(
 			}
 		},
 		async deleteApiKey() {
+			this.$telemetry.track('User clicked delete API key button', {
```
Comment (mutdmour) [inline packages/editor-ui/src/views/SettingsApiView.vue:123]: nitpick - feels like these should go in a finally block after api call

File: packages/editor-ui/src/views/SettingsApiView.vue [hunk 3]
```diff
@@ -105,6 +106,10 @@ export default mixins(
 		},
 		async createApiKey() {
 			this.loading = true;
+			this.$telemetry.track('User clicked create API key button', {
+				user_id: this.$store.getters['users/currentUserId'],
+				instance_id: this.$store.getters.instanceId,
```
Comment (mutdmour) [inline packages/editor-ui/src/views/SettingsApiView.vue:111]: applies here and below... you don't need user id or instance id. they are included by default

File: packages/editor-ui/src/components/WorkflowActivator.vue (hunk 1 above)
Comment (alexgrozav) [reply packages/editor-ui/src/components/WorkflowActivator.vue]: Thank you for letting me know. Removed the `user_id` and `instance_id` fields.

File: packages/editor-ui/src/views/SettingsApiView.vue (hunk 3 above)
Comment (alexgrozav) [reply packages/editor-ui/src/views/SettingsApiView.vue]: Thank you. Removed!

**Review by mutdmour — APPROVED** [review]

**Pull request merged**